pkg/skaffold/kubernetes/manifest/transformations.go

```diff
@@ -35,3 +36,18 @@ var manifestTransforms []ManifestTransform
 func AddManifestTransform(newTransform ManifestTransform) {
 	manifestTransforms = append(manifestTransforms, newTransform)
 }
+
+func GetManifestTransforms() []ManifestTransform {
+	return manifestTransforms
+}
+
+func ApplyTransforms(manifests ManifestList, builds []build.Artifact, insecureRegistries map[string]bool, debugHelpersRegistry string) (ManifestList, error) {
+	var err error
```

Comment:
nit: suggest defining a multiError to catch all bad manifestTransforms. (so as users don't need to run multiple times if the manifest has more than one error. )

Reply:
fair suggestion, but this also isn't new code so i'll leave this change out of this PR